File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/com/github/asyncmc/mojang/api/scala/gatling/model/UploadSkinRequest.scala

package com.github.asyncmc.mojang.api.scala.gatling.model

import java.io.File

case class UploadSkinRequest (
    _model: Option[SkinModel],
    /* The skin image in PNG format */
    _file: File
)
object UploadSkinRequest {
    def toStringBody(var_model: Object, var_file: Object) =
        s"""
        | {
        | "model":$var_model,"file":$var_file
        | }
        """.stripMargin
}

File: generated-sources/php-slim/mojang-api/lib/com/github/asyncmc/mojang/api/php/slim/model/UploadSkinRequest.php
<?php
/**
 * UploadSkinRequest
 */
namespace com.github.asyncmc.mojang.api.php.slim.server\com.github.asyncmc.mojang.api.php.slim.model;

/**
 * UploadSkinRequest
 */
class UploadSkinRequest
{
    
    /** @var \com.github.asyncmc.mojang.api.php.slim.server\com.github.asyncmc.mojang.api.php.slim.model\SkinModel $model */
    private $model;
    
    /** @var \SplFileObject $file The skin image in PNG format*/
    private $file;
}
